plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -866,9 +879,11 @@ fun MapPluginExtensionsDelegate.moveBy(
  * @param first The first pointer to rotate on
  * @param second The second pointer to rotate on
  * @param animationOptions Transition options (animation duration, listeners etc)
+ *
+ * @return [Cancelable] animator set object or null if associated map object was garbage collected.
  */
 fun MapPluginExtensionsDelegate.rotateBy(
   first: ScreenCoordinate,
   second: ScreenCoordinate,
   animationOptions: MapAnimationOptions? = null
-) = cameraAnimationsPlugin { rotateBy(first, second, animationOptions) }
\ No newline at end of file
+) = cameraAnimationsPlugin { rotateBy(first, second, animationOptions) } as Cancelable
```

Comment:
do we need to do a cast as Cancelable here?

Reply:
Yes, we do, we use abstraction with `Any?` under the hood.